modules/developer_manual/pages/general/backporting.adoc

```diff
@@ -46,23 +50,33 @@ based off of the branch that you wish to backport to. However, doing so
 can involve a number of manual steps. To reduce the effort and time
 involved, use the script below instead.
 
-=== Backporting Script
+=== Backporting Notes
+
+NOTE: The script relies on a recent version of `grep`. macOS users may find on
+their system a version provided from Apple which is outdated and lacking needed
+options. Use {homebrew-url}[homebrew] to install a recent version of `grep`.
 
 NOTE: The script uses `curl` and the `jq` (lightweight and flexible
 command-line JSON processor) package. Please install them before first usage.
-Please see this https://stedolan.github.io/jq/download/[link] for installation
+Please see this {json-processor-url}[link] for installation
 details of `jq` covering various OS.
 
 NOTE: This script uses the github API. For unauthenticated requests, the rate
 limit allows for up to 60 requests per hour. Unauthenticated requests are
 associated with the originating IP address, and not the user making requests.
-Please see this https://developer.github.com/v3/#rate-limiting[link] for more
+Please see this {rate-limit-url}[link] for more
 information about github rate limiting.
 
+NOTE: The script requires that you have checked out the branch containing the
+merge-sha. The script will not proceed if either the merge-sha is not present or the branch containing the
```

Comment:
I suggest changing "merge-sha" to "merge SHA1 hash".

Reply:
done